android/books/src/main/java/kmp/android/books/navigation/BooksDestination.kt

```diff
@@ -0,0 +1,35 @@
+package kmp.android.books.navigation
+
+import android.os.Bundle
+import androidx.navigation.NamedNavArgument
+import androidx.navigation.NavType
+import androidx.navigation.navArgument
+import kmp.android.shared.navigation.Destination
+import kmp.android.shared.navigation.RootDestination
+
+object BooksDestination : RootDestination {
```

Comment:
I was hesitant at first but I actually like the destination definition in the feature modules. Looks very clean. It may hurt discoverability a little but that's probably fine. I just wonder, have you ever worked with Android dynamic modules? Would this work with them? 🤔 I've never used them but in my mind this would not work well with them.

Reply:
I have not used the dynamic modules yet either, I suppose if we used them on some project we would have to modify this navigation solution or not use it at all.